website_forum_censored/models/forum_censored_phrase.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ForumCencoredPhrase(models.Model):
+    _name = 'forum.censored.phrase'
+
+    phrase = fields.Char('Phrase', required=True)
```

Comment:
you should validate this. It is easy to create a phrase which is not a valid regexp, which will break with an obscure message on the creation of a forum post. I'd rather have a python constraint on the `phrase` field which makes sure the expression can be compiled.